packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -314,6 +314,8 @@ export abstract class AbstractPureGraphManager {
     lambda: RawLambda,
     runtime: Runtime,
     clientVersion: string,
+    parameters: (string | number | boolean)[],
+    hashStrings: boolean,
```

Comment:
Let's put `hashStrings` in an `options` field? and also let's document that a bit since the feature doesn't get reflected very clearly with the name now

Reply:
Also, I think at this level, we should call this `anonymizeGeneratedData`